core/sts-3.3.md

```diff
@@ -0,0 +1,129 @@
+---
+title: IRCv3.3 Strict Transport Security
+layout: spec
+copyrights:
+  -
+    name: "Attila Molnar"
+    period: "2015"
+    email: "attilamolnar@hush.com"
+---
+
+## Description
+
+Strict Transport Security (STS) is a mechanism which allows servers to
+communicate that complying clients should only connect to them over a secure
+connection.
+
+The policy is communicated to the client via the `sts` capability and should
+be processed by the client at capability negotiation time.
+
+The value of the capability specifies the duration during which the client
+MUST only connect to the server securely (using TLS, aka. SSL), and the port
+number to use for it.
+
+Secure connections must use a TLS server certificate that is signed with a
+certificate from a reputable CA chain, using either the OS trust root or a
+list such as Mozilla's.
```

Comment:
@moparisthebest that's all out of scope of STS.

The idea is to make this paragraph less specific to *allow* clients to use alternative methods if they consider them better than just traditional CA validation. Which can be a controversial topic, but there's no need to get into those details.